lib/Checkout/Payments/Four/Request/Source/Apm/RequestPayPalSource.php

```diff
@@ -0,0 +1,14 @@
+<?php
+
+namespace Checkout\Payments\Four\Request\Source\Apm;
+
+use Checkout\Common\PaymentSourceType;
+use Checkout\Payments\Four\Request\Source\AbstractRequestSource;
+
+class RequestPayPalSource extends AbstractRequestSource
+{
+    public function __construct()
```

Comment:
There are no properties at all?

Reply:
Based on the provided documentation seems not.
https://www.checkout-docs-private-beta.com/docs/four/paypal#Redirect_the_customer